### DISCUSSION TITLE
Insert exit code / return value / $? in prompt (only if > 0)

### DISCUSSION THREAD

**Discussion by luckman212** [starship/starship#4649] [Q&A] [answered]
I know you can use `[character]` to change the prompt char when the previous command fails in some way.

```
[character]
success_symbol = "[✔](green)"
error_symbol = "[✘](red)"
```

Is it possible to also print the exit code from the failed command as part of the prompt?

**Comment by davidkna** [comment, marked as answer]
Please try enabling the `status` module (`starship config status.disabled false`)

**Comment by luckman212** [comment]
Thanks @davidkna - 🫢 embarrassed I didn't see this in https://starship.rs/config/#status
